src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -58,7 +58,7 @@ def _get_default_install_location(exe_name):
 register_cli_argument('acs', 'name', arg_type=name_arg_type, help='ACS cluster name', completer=get_resource_name_completion_list('Microsoft.ContainerService/ContainerServices'))
 register_cli_argument('acs', 'resource_group', arg_type=resource_group_name_type)
 
-register_cli_argument('acs', 'orchestrator_type', **enum_choice_list(ContainerServiceOchestratorTypes))
+register_cli_argument('acs', 'orchestrator_type', options_list=('--orchestrator_type', '-t'), **enum_choice_list(ContainerServiceOchestratorTypes))
```

Comment:
I think you're right. We can't do this breaking change. Ideally, we can do multiple aliases for one parameter, however this would further confusing the namespace. Let's keep the name for now. We can think of a way to introduce obsoleted parameter if necessary.

In addition, the parameter should be registered as `--orchestrator-type` instead of `--orchestrator_type`.